webtester-core/src/test/java/info/novatec/testit/webtester/pageobjects/UnorderedListTest.java

```diff
@@ -0,0 +1,107 @@
+package info.novatec.testit.webtester.pageobjects;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.Matchers.contains;
+
+import java.util.LinkedList;
+
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+
+import info.novatec.testit.webtester.AbstractPageObjectTest;
+import info.novatec.testit.webtester.api.exceptions.WrongElementClassException;
+
+public class UnorderedListTest extends AbstractPageObjectTest{
+
+    @Mock
+    ListItem item1;
+
+    @Mock
+    ListItem item2;
+
+    @Spy
+    java.util.List<ListItem> itemList = new LinkedList<ListItem>();
+
+    @InjectMocks
+    UnorderedList cut;
+
+    /* empty */
+
+    @Test
+    public final void testThatListIsEmpty_EmptyList() {
+        assertThat(cut.isEmpty(), is(true));
+    }
+
+    @Test
+    public final void testThatListIsEmpty_FilledList() {
+        addBothItems();
+        assertThat(cut.isEmpty(), is(false));
+    }
+
+    /* number of items */
+
+    @Test
+    public final void testThatListHasCorrectNumberOfItems_EmptyList()  {
+        assertThat(cut.getNumberOfItems(), is(0));
+    }
+
+    @Test
+    public final void testThatListHasCorrectNumberOfItems_FilledList() {
+        addBothItems();
+        assertThat(cut.getNumberOfItems(), is(2));
+    }
+
+    /* get item */
+
+    @Test
+    public final void testThatItemCanBeRetrievedCorrectly() {
+        addBothItems();
+        assertThat(cut.getItem(0).equals(item1), is(true));
```

Comment:
`assertThat(cut.getItem(0).equals(item1), is(true));` >> `assertThat(cut.getItem(0), is(item1));`